testbed/person_data/data.json

```diff
@@ -0,0 +1,20 @@
+{
+    "People": [
```

Comment:
This data format feels weird to me? Why are we combining a Family with a Person? I think it would make more sense to have the model work on something more realistic / structured. In most json, whenever you have an array it is almost always an array of one thing; if you want multiple things, you would use key value pairs instead. Not the biggest since obviously this is just a benchmark, but I do think it makes more sense to give it something it has not only seen a lot more but also what it will see.

Reply:
It's true it's a weird data format and that's probably why that's the most challenging part of the test for the model. Realism is definitely good (i.e. lining it up with something similar to what it will see) but we don't want to just give it tests that it has seen before and could have memorized. Since the model can see all of the data it should be able to write code to parse arbitrary formatting.
I'm on the fence about the current data format and would like @biobootloader to chime in once he's read it.

Reply:
I think my major issue with the current format is that it requires us to prompt engineer it to get anything correct; the entire point of the benchmarks is so that we can see if a specific prompt does better or worse than the last one. With per benchmark prompt engineering I feel like that throws off the measurements a bit and makes it more difficult to tell if a prompt is actually better or worse

Reply:
We could certainly use one of the more difficult prompts with fewer hints. I think that might be better than just giving it an easier data format though. Then we could see if the general prompt improves performance at all as opposed to if the benchmark isn't a challenge at all.

Reply:
I think the ideal benchmark here would be something where the prompt is fairly straightforward, i.e. just what a person would like to be able to ask it to do, without hints it should be able to gather from looking at the data format. And performance should be something like 25%-50%, as that is both 1) high enough that we can see improvements quickly and 2) low enough that we have something to improve on

If simplifying/standarizing the data format is the way to do that, then lets do it